### DISCUSSION TITLE
Is it possible to make this work on Viomi v7?

### DISCUSSION THREAD

**Discussion by 8times4** [rand256/valetudo#397] [Q&A]
Hey Everyone, 

Is there someone who was able to get this working on a Viomi v7? 
Regular Valetudo from Hypfer works, but just replacing the binary does not. 
The conf file does not really work since it's probably meant to be for debian based robots, but I'm just wondering if anyone made progress with this. 

Thanks,
Mark

**Comment by pidator** [comment]
Duplicte of https://github.com/rand256/valetudo/issues/400
@rand256 answered the question there.
